.github/workflows/update-links.yml

```diff
@@ -63,10 +63,14 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GH_BOT_TOKEN }}
 
-      - name: Deploy to Production
```

Comment:
I should've asked, but I assume the removal of this step is intentional?

Reply:
aghhh thanks, #356